ci/docker/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
```

Comment:
Added new script to local build image.

https://github.com/avito-tech/avito-android/pull/1335/files#r805345515

Reply:
Don't get what `to local build image` means

Reply:
I mean this scenario:

1. I've made changes in image X and want to check them locally.
2. I run `./build.sh X` and get this X image.
3. Repeat 1-2 if needed.
4. Push changes